services/src/contexts/mod.rs

```diff
@@ -430,7 +427,7 @@ mod tests {
 
         let exe_ctx = ctx.execution_context().unwrap();
 
-        let model_path = PathBuf::from("xgboost/s2_10m_de_marburg/model.json");
+        let model_path = PathBuf::from("xgboost/s2_10m_de_marburg/");
```

Comment:
this should be a uuid/ModelId now right?

Reply:
1168be2810c99a6bbee7b5b4d26596dee1373a5a